testhelpers/testhelpers.go

```diff
@@ -409,3 +410,14 @@ func checkResponseError(r io.Reader) error {
 
 	return nil
 }
+
+func SetUpMockServer(t *testing.T, repo string, statusCode, failedCount int) (*MockServer, string) {
```

Comment:
What do you think about moving this function to `mock_registry.go`?

Reply:
I thought about it at first, but then I was not sure, and because of this testhelpers are other methods reuse in other places I decided to put it here. But, I think it will not harm anyone if we move it to `mock_registry.go`